services/motion/motion_helpers_test.go

```diff
@@ -0,0 +1,94 @@
+package motion_test
```

Comment:
We've largely stuck to the pattern of `file.go` having functions that are tested in `file_test.go` and since the helper lives in `motion.go` I think we should not make a new file and put it in that one as well.  Else, move the original function in to its own file

Reply:
I did this b/c this test needs a mocked motion service, and inject depends on the motion package. Adding this test in the current motion_test.go would cause a circular dependency. 

I think the solution is to change motion_test.go to be in the `motion_test` package & move any private functions being tested into `motion_helpers_test.go` but I'd like that to be something we do after this PR.